compiler/env/OMRVMEnv.hpp

```diff
@@ -108,6 +108,13 @@ class OMR_EXTENSIBLE VMEnv
    bool ensureOSRBufferSize(TR::Compilation *comp, uintptrj_t osrFrameSizeInBytes, uintptrj_t osrScratchBufferSizeInBytes, uintptrj_t osrStackFrameSizeInBytes) { return false; }
    uintptrj_t thisThreadGetOSRReturnAddressOffset(TR::Compilation *comp) { return 0; }
 
+   uintptrj_t thisThreadGetGSIntermediateResultOffset(TR::Compilation *comp) { return 0; }
+   uintptrj_t thisThreadGetConcurrentScavengeActiveByteAddressOffset(TR::Compilation *comp) { return 0; }
+   uintptrj_t thisThreadGetEvacuateBaseAddressOffset(TR::Compilation *comp) { return 0; }
+   uintptrj_t thisThreadGetEvacuateTopAddressOffset(TR::Compilation *comp) { return 0; }
+   uintptrj_t thisThreadGetGSOperandAddressOffset(TR::Compilation *comp) { return 0; }
+   uintptrj_t thisThreadGetGSHandlerAddressOffset(TR::Compilation *comp) { return 0; }
```

Comment:
Could we please document these new functions in a way such that any potential downstream projects know exactly how to override and implement the respective functions? i.e. semantically what does each value represent?

Reply:
Added comments